consensus/src/epoch_manager.rs

```diff
@@ -806,7 +806,7 @@ impl EpochManager {
                 end_batch_ms: 500,
                 max_batch_bytes: 100000,
                 batch_request_timeout_ms: 10000,
-                max_batch_expiry_round_gap: 100,
+                max_batch_expiry_round_gap: 20,
```

Comment:
this param isn't used atm I think anyway.

Reply:
or I guess it was used in BatchReader (which we commented), but also used to set the actual expiry?